test_tsp.py

```diff
@@ -220,3 +226,26 @@ def test_fetch_timegraph_tree(self, kernel):
         assert response.model.model_type == response.model.model_type.TIME_GRAPH_TREE
         self._delete_experiments()
         self._delete_traces()
+
+    def test_fetch_extensions_none(self):
+        response = self.tsp_client.fetch_extensions()
+        assert response.status_code == 200
+        assert not response.model.extension_set
+
+    def test_post_extension(self, extension):
+        response = self.tsp_client.post_extension(extension)
+        assert response.status_code == 200
+
+        response = self.tsp_client.fetch_extensions()
+        assert len(response.model.extension_set) == 1
+
+        response = self.tsp_client.delete_extension(self.name)
+        assert response.status_code == 200
+
+    def test_posted_extension_deleted(self, extension):
+        response = self.tsp_client.post_extension(extension)
```

Comment:
Is it worth to add an `assert response.status_code == 200` here?

Reply:
> Is it worth to add an `assert response.status_code == 200` here?

Based on [1,2] from this class docstring, test_post_extension above should already cover this specific assert or case. So I thought of not cluttering this differently scoped test with it. Should this post fail for some reason, then the delete next below should fail. If posting a new extension fails in general during that time, then test_post_extension (above) should fail, as expected.

So for such setup steps, I try to minimize their assertions, so the test code may focus on its scoped assert steps. We may of course add assertions for potentially problematic steps as they show. I tried to summarize that approach in [1,2+] (cf. Class docstring further above).